activesupport/lib/active_support/railtie.rb

```diff
@@ -7,6 +7,13 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.eager_load_namespaces << ActiveSupport
 
+    initializer "active_support.set_authenticated_message_encryption_as_default" do |app|
```

Comment:
@kaspth 
Whenever, I make this change, Travis CI fails, because of sessions_test.rb which outputs:
1) Failure:
ApplicationTests::MiddlewareSessionTest#test_session_upgrading_from_AES-CBC-HMAC_encryption_to_AES-GCM_encryption [test/application/middleware/session_test.rb:351]:
Expected: "1"
  Actual: ""